docs/guides-agent-app.md

```diff
@@ -0,0 +1,433 @@
+
+    - And the third (`true`) specifies whether the contract should check if a vote already has enough support to be executed. By executed we mean that even if everyone else voted against, the vote would still be approved. If that's the case, the vote is executed and immediately closed. `true` means check, `false` means don't check.
+
+<p align="center">
+   <img width="400" src="/docs/assets/agent-guide/agent-4.png">
+   <em>It's easy to find vote ids from the UI: see the red circle.</em>
+</p>
+
+## 4. Check permissions
+
+If you rerun the command:
+
+`dao apps <your organization name> --all --environment aragon:rinkeby`
+
+You should see that your Agent app has been added to the bottom of the App table and that the Permissionless app table is now empty.
+
+| App  | Proxy address | Content |
+| ------------- | ------------- | ------------ |
+| kernel@vundefined  | 0xa25fb31870bc492d450012ae32dafa72af9e82c3 | (No UI available) |
+| acl@vundefined | 0xfefb0cdb7a1fac257815d52ba82776f98dc70205 | (No UI available) |
+| evmreg@vundefined | 0x9087db02300ef24b116daf0426b6ba22b28a0c79 | (No UI available) | 
+| voting@v2.0.4 | 0x15a102f80ea3b1bd585a044e9b3c39a84c5f44e5 | ipfs:QmPjWU51opgTVnXwAhYAWasL2CaiYHqy2mXdXtzqfC8sKx |
+| vault@v3.0.1   | 0x952a18185da912984e0bc8a830ba98f8151976af | ipfs:QmeMabCnkA5BtTTszqqRztYKCXZqE9VQFH4Vx7dY9ue2nA |
+| finance@v2.0.5    | 0x4171f7ac1a4606b93093e8648e2f9a16c59cf3b1 | ipfs:QmeMLs4jHya89khHVSubLaao9cZW6ELZUoYPHkwCUwKBH7 |
+| token-manager@v2.0.3  | 0xbf07e1c74a72aa60df3ddf3115d15575d27e61e1 | ipfs:Qmb9Bv3J9AuXD5auY1WNwiJeohnYRhyso7XMULs7EZ8eTG |
+| **0x9ac98dc5f995bf0211ed589ef022719d1487e5cb2bab505676f0d084c07cf89a** | 0x843bfa21a040e742ec32b8f6991e182d9655af21 | ipfs:QmfNaBuQsaKE8at2ce9k2FU9dKs16WQqg4RPUHSNik1z9e |
+
+
+| Permissionless app   | Proxy address  |
+| ------------- | ------------- |
+
+This confirms that the Agent app has been assigned permissions and is now an app in the DAO.
+
+Note that **0x9ac98dc5f995bf0211ed589ef022719d1487e5cb2bab505676f0d084c07cf89a** is just the general identifier for the version of Agent app you have installed.
+
+You can also verify that permissions have been set properly through the UI:
+
+1. Click on the **Permissions** menu option in the left panel. You should see the Agent app at the end of the second row. Click on it.
+
+<p align="center">
+   <img width="800" src="/docs/assets/agent-guide/agent-5.png">
+</p>
+
+2. Under **Actions available on this app** should see that the Voting app now manages who has the ability to execute actions for the Agent app. Scroll down.
+
+<p align="center">
+   <img width="800" src="/docs/assets/agent-guide/agent-6.png">
+</p>
+
+3.  Under **Permissions set on this app** you should see that Voting app is also allowed to execute actions on behalf of the Agent app.
+
+<p align="center">
+   <img width="800" src="/docs/assets/agent-guide/agent-7.png">
+</p>
+
+# Use cases
```

Comment:
As we discussed earlier, this section will benefit of some images to make it easy to digest the technical language 😅

Reply:
I think I've addressed this for the first three steps with the new images.

For the last two,  I'm still waiting for the ABI's to be fetched...